fxmanifest.lua

```diff
@@ -35,3 +35,5 @@ files {
 dependencies {
 	'qbr-core'
 }
+
+lua54 'yes
```

Comment:
Missing a '
`lua54 'yes`
should be
`lua54 'yes'`

Reply:
This Been Corrected